src/frontend/src/lib/canisters/backend.canister.ts

```diff
@@ -94,4 +100,66 @@ export class BackendCanister extends Canister<BackendService> {
 			credential_spec: credentialSpec
 		});
 	};
+
+	btcAddPendingTransaction = async ({
+		txId,
+		network,
```

Comment:
nitpick: you might not need to descruct everything, you might be able to use `...rest`

Reply:
Done.